test/Interpreter/EvalSpec.hs

```diff
@@ -275,3 +289,161 @@ spec = do
     |]
 
     testProgramm map (List [Integer 2, Integer 3, Integer 4])
+
+    -- test for calling a non-existant function
```

Comment:
I'd say `test for trying to call non-function or getting a length of non-list` or something similar.